pkg/pushsync/pushsync_test.go

```diff
@@ -211,102 +211,6 @@ func TestReplicateBeforeReceipt(t *testing.T) {
 	}
 }
 
-func TestFailToReplicateBeforeReceipt(t *testing.T) {
```

Comment:
why is the test removed? we still do replication no?

Reply:
This test is the same as one before it, the only difference is that this test checks that the replication fails because the replication node is out of depth. Since this requirement no longer exists, this test is invalid.